slamon_afm/models.py

```diff
@@ -7,10 +8,24 @@
     TypeDecorator
 from sqlalchemy.orm import relationship, backref
 from sqlalchemy.orm.exc import NoResultFound
-from sqlalchemy import func
+from sqlalchemy import func, event, case, literal_column
 
 db = SQLAlchemy()
 
+from slamon_afm.stats import statsd
+
+
+class TaskException(Exception):
+    task = None
+
+    def __init__(self, *args, task=None):
```

Comment:
I managed to silence the linter warnings and then get a correct behavior with this:

```python
class TaskException(Exception):
    def __init__(self, *args, **kwargs):
        super(TaskException, self).__init__(*args)
        self.task = kwargs.get('task')
```